lib/subcommands/update_versions.py

```diff
@@ -227,17 +229,20 @@ def run(CONF):
     for pkg in pm.packages:
         pkg.lock()
         pkg_version = Version(pkg)
-        pkg_version.update(updater_name, updater_email)
+        updates_available = pkg_version.update(updater_name, updater_email)
         pkg.unlock()
 
     packages_names = discover_packages()
     METAPACKAGE_NAME = "open-power-host-os"
     packages_names.remove(METAPACKAGE_NAME)
 
-    update_metapackage(
-        versions_repo, distro, METAPACKAGE_NAME, packages_names,
-        updater_name, updater_email)
-    update_versions_in_readme(versions_repo, distro, packages_names)
+    if updates_available: 
```

Comment:
The commit should not be created if there are no updates. You could either include the rest of the function in the `if` or place this right before the call to `discover_packages()`:
```
if not updates_available:
    LOG.info("No updates in packages versions.")
    continue
```
I prefer the former to avoid indenting all those lines.